site/examples/kuard-httpproxy.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
```

Comment:
Where is this file referenced from?

Reply:
Should be site/docs/master/deploy-options.md ... which appears to be missing from the latest squash. Oops (will correct).

Reply:
Ping. Would you please rebase and squash to one commit then reply so we know to review again. Thanks Dave